pkg/Library/Mediapipe/DetectRaisedHandNode.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */
+export const DetectRaisedHandNode = {
+  $meta: {
+    id: 'DetectRaisedHandNode',
+    displayName: 'Detect Raised Hands',
+    description: 'Look for raised hands in Mediapipe Pose Detection results.',
+    category: 'Mediapipe',
+  },
+  $stores: {
+    pose: {
+      $type: 'PoseData',
+      connection: true,
```

Comment:
i will do it in a follow up PR, but we should (try?) stop putting `connection: true` in nodes, it _should_ work

Reply:
Ok, heard Chef!